content/docs/0.8.x/operate/k8s_support/index.md

```diff
@@ -21,13 +21,13 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 
 | Keptn Version /<br>Installation                           | Kubernetes  | AKS                       | EKS         | GKE           | OpenShift   | K3s         | Minishift               |
 |-----------------------------------------------------------|:-----------:|:-------------------------:|:-----------:|:-------------:|:-----------:|:-----------:|:------------------------|
-| **0.8.x** / <br>Control & Execution plane<br>*see: (3)*   | 1.19 - 1.16 | 1.18 - 1.15<br>*see: (1)* | 1.18 - 1.16 | 1.18 - 1.15   | 4, 3.11     | 1.19 - 1.16 | 1.34.2<br>(K8s: 1.11)   |
-| **0.8.x** / <br>Control plane                             | 1.20 - 1.16 | 1.18 - 1.15<br>*see: (1)* | 1.18 - 1.16 | 1.18 - 1.15   | 4, 3.11     | 1.20 - 1.16 | 1.34.2<br>(K8s: 1.11)   |
+| **0.8.x** / <br>Control & Execution plane<br>*see: (3)*   | 1.21 - 1.16 | 1.18 - 1.15<br>*see: (1)* | 1.18 - 1.16 | 1.19 - 1.15   | 4, 3.11     | 1.21 - 1.16 | 1.34.2<br>(K8s: 1.11)   |
+| **0.8.x** / <br>Control plane                             | 1.21 - 1.16 | 1.18 - 1.15<br>*see: (1)* | 1.18 - 1.16 | 1.19 - 1.15   | 4, 3.11     | 1.21 - 1.16 | 1.34.2<br>(K8s: 1.11)   |
 
 **Remarks:**
 
 * (1): AKS with K8s version before 1.15 might fail (see: [#1429](https://github.com/keptn/keptn/issues/1429)), due to a known AKS issue: [#69262](https://github.com/kubernetes/kubernetes/issues/69262)
-* (2): GKE and EKS did not provide K8s 1.19 clusters for testing (checked on: 3rd March, 2021)
+* (2): EKS did not provide K8s 1.19 clusters for testing (checked on: 3rd March, 2021)
```

Comment:
Can you please add the reference `see: (2)` to the two cells: EKS > `1.18 - 1.16`

Reply:
done